microcosm_flask/swagger/definitions.py

```diff
@@ -121,11 +122,13 @@ def build_path(operation, ns):
     try:
         return ns.url_for(operation, _external=False)
     except BuildError as error:
+        # we are missing some URI path parameters
         uri_templates = {
             argument: "{{{}}}".format(argument)
             for argument in error.suggested.arguments
         }
-        return ns.url_for(operation, _external=False, **uri_templates)
+        # flask will sometimes try to quote '{' and '}' characters
```

Comment:
I'm not enabling swagger definitions for Alias endpoints by default, but they will work to a first order as long as we use `unquote` here. Otherwise, the substitution that happens here will break for some obscure Flask reason.

NB: the path for an Alias to Foo is `/foo/{foo_name}`, but Flask's `url_for()` expects us to substitute a *real* value for `foo_name` into this path. So we substitute the string literal `"{foo_name}"`... this works except somehow if the literal doesn't contain the usual "uuid:" qualifier, Flask decides to escape the curly braces.

Ugh.